site/docs/plugins/auto-retry.md

```diff
@@ -1,16 +1,13 @@
 # Retry API Requests (`auto-retry`)
 
-> Consider using the [throttler plugin](./transformer-throttler.md) instead.
-
 This plugin is an [API transformer function](../advanced/transformers.md), which means that it let's you intercept and modify outgoing HTTP requests on the fly.
 More specifically, this plugin will automatically detect if an API requests fails with a `retry_after` value, i.e. because of rate limiting.
 It will then catch the error, wait the specified period of time, and then retry the request.
 
-::: warning Be Gentle With the Bot API Server
-Telegram is generously providing information about how long your bot must wait before the next request.
-Using the `auto-retry` plugin will allow your bot to perform better during load spikes, as the requests will not simply fail because of the flood limit.
-However, **auto-retry should not be used** if you want to avoid hitting rate limits on a regular basis.
-If you regularly cross the threshold of how many requests you may perform, Telegram may take measures such as restricting or banning your bot.
+:::tip Flood Control
+Telegram will let you know if you send messages too fast.
+This is an important measure for flood control, as it makes sure that your bot does not put Telegram under too much load.
+Using this plugin is important because if you forget to respect 429 errors, Telegram may ban your bot.
```

Comment:
```suggestion
Using this plugin is important because if you forget to respect [429 errors](../resources/faq.md#_429-too-many-requests-retry-after-x), Telegram may ban your bot.
```

Please, remind me to make separate PQ for this.